admin/resources/js/stories/TextArea.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { Meta, Story } from "@storybook/react";
+import Form from "../components/form/Form";
+import Submit from "../components/form/Submit";
+import TextArea, { TextAreaProps } from "../components/form/TextArea";
+
+export default {
+  component: TextArea,
+  title: "Form/TextArea",
+} as Meta;
+
+const TemplateTextArea: Story<TextAreaProps> = (args) => {
+  return (
+    <div style={{ width: "20rem" }}>
+      <Form onSubmit={action("Submit Form")}>
+        <TextArea {...args} />
+        <Submit />
+      </Form>
+    </div>
+  );
+};
+
+export const BasicTextArea = TemplateTextArea.bind({});
+
+BasicTextArea.args = {
+  id: "description",
+  context: "Additional context about this field.",
+  label: "Description",
+  name: "description",
+  rules: { required: "This field is required", maxLength: 100 },
```

Comment:
I don't see `maxLength` value having an effect on submit or when typing, is it supposed to show an error on submit?

Reply:
Again, I forgot to add a message to the maxLength rule.